Vagrantfile

```diff
@@ -4,20 +4,36 @@ load "settings.sh"
 
 Vagrant.configure("2") do |config|
   
-  config.vm.box = "fujimakishouten/debian-buster64"
+  config.vm.box = VB_IMAGE
 	
-  config.vm.network "public_network", ip: LAN_IP, bridge: INTERFACE, netmask: "255.255.255.0"
+  if STANDALONE.downcase == "yes"		
+    config.vm.network "public_network", ip: LAN_IP, netmask: "255.255.255.0", bridge: LAN_IF
+  else
+    config.vm.network "public_network", auto_config: true, bridge: LAN_IF
+  end
+  
+  if Vagrant.has_plugin?("vagrant-cachier")
+    config.cache.scope = :box
+    config.vm.network :private_network, ip: "10.0.2.52"
+    config.cache.synced_folder_opts = {
+      type: :nfs,
+      mount_options: ['rw', 'vers=3', 'tcp', 'nolock']
+    }
+   end
   
   config.vm.provider "virtualbox" do |virtualbox|
 	  # Enable promiscuous mode
   	  virtualbox.customize ["modifyvm", :id, "--nicpromisc2", "allow-all"]
+  	  virtualbox.memory = VB_RAM
+   end
+  
+  config.vm.provider "virtualbox" do |vb|
+    vb.gui = true
+    vb.memory = "1024"
```

Comment:
Why do you need this?

Reply:
I mean why do you need this line of code.

Reply:
In the recent [commit](https://github.com/docker-scripts/dev--LTSP/pull/93/commits/2452f8202195c8089eeb537aff7732e59ce78c93) we added `xfce` gui. This line of code will open a virtualbox windows for the ltsp-server. It will allow us to use gui programs like `epoptes`, etc.

If we removed it the vagrant box will start with default mode(headless). And we will not be able to see gui despite of being installed.

I you are talking about `vb.memory = "1024"` that line can be removed without any issues.(I added it because it was present by default in `Vagrantfile` template)

Reply:
Yes, I am talking about the memory line. What is it used for?

Reply:
It is used to Customize the amount of memory on the VM. Should be removed.